ee/tabby-ui/app/(dashboard)/(logs)/jobs/components/job-detail.tsx

```diff
@@ -40,33 +42,81 @@ export default function JobRunDetail() {
     }
   }, [currentNode])
 
+  console.log(currentNode)
   return (
     <>
       {fetching ? (
         <ListSkeleton />
       ) : (
         <div className="flex flex-1 flex-col items-stretch gap-2">
-          <JobsTable jobs={edges?.slice(0, 1)} shouldRedirect={false} />
-          <Tabs defaultValue="stdout" className="flex flex-1 flex-col">
-            <TabsList className="grid w-[400px] grid-cols-2">
-              <TabsTrigger value="stdout">
-                <IconTerminalSquare className="mr-1" />
-                stdout
-              </TabsTrigger>
-              <TabsTrigger value="stderr">
-                <IconAlertTriangle className="mr-1" />
-                stderr
-              </TabsTrigger>
-            </TabsList>
-            <div className="flex flex-1 flex-col">
-              <TabsContent value="stdout">
-                <StdoutView value={currentNode?.stdout} />
-              </TabsContent>
-              <TabsContent value="stderr">
-                <StdoutView value={currentNode?.stderr} />
-              </TabsContent>
-            </div>
-          </Tabs>
+          {currentNode && (
+            <>
+              <h1 className="text-4xl font-semibold tracking-tight first:mt-0">
+                {currentNode.job}
+              </h1>
+              <div className="flex gap-16 pb-6 pt-2">
+                <div>
+                  <p
+                    className={`text-${findColorByExitCode(
```

Comment:
Should not construct class names dynamically. On the server I'm running locally, this color classname does not take effect.